lists-service/src/main/java/au/org/ala/listsapi/controller/ElasticUtils.java

```diff
@@ -311,7 +278,7 @@ public static void restrictFields(String searchQuery, HashSet<String> restricted
                         .path("properties")
                         .scoreMode(ChildScoreMode.Avg)
                         .query(nq -> nq.bool(nb -> {
-                            nb.must(m1 -> m1.term(t -> t
+                            nb.should(m1 -> m1.term(t -> t
```

Comment:
this is actual fix for issue - other changes were to remove the duplicated code.